css-layout-classes/styles.css

```diff
@@ -0,0 +1,44 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: none;
+  font-family: sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+img {
+  width: 100%;
+}
+
+.row {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.column-two-thirds {
+  flex-basis: calc(200% / 3);
+}
+
+.column-one-fourth {
+  display: flex;
```

Comment:
This should not be necessary. Column classes only have `flex-basis` declarations, nothing else.

Reply:
![image](https://user-images.githubusercontent.com/117887912/206594136-9575dcad-623e-48c1-90ce-a54f6e73f239.png)